app/models/section.rb

```diff
@@ -42,7 +38,7 @@ def deep_copy(**options)
     copy.modifiable = options.fetch(:modifiable, self.modifiable)
     copy.phase_id = options.fetch(:phase_id, nil)
     copy.save!(validate: false)  if options.fetch(:save, false)
-    options[:section_id] = id
+    options[:section_id] = copy.id
```

Comment:
@briri @xsrust Please have a look at this suspected bug and confirm. Thanks.

Reply:
I'm also concerned about bad data and will check our services to see if anything needs cleaned up.